providers/implementations/signature/rsa_sig.c

```diff
@@ -547,6 +547,33 @@ static void free_tbuf(PROV_RSA_CTX *ctx)
     ctx->tbuf = NULL;
 }
 
+#ifdef FIPS_MODULE
+static int rsa_pss_saltlen_check_passed(PROV_RSA_CTX *ctx, const char *algoname)
+{
+    int mdsize = rsa_get_md_size(ctx);
+    int saltlen = rsa_pss_compute_saltlen(ctx);
```

Comment:
PR #25085 has some fixes in it for rsa_pss_compute_saltlen that make it return -1.
SO I wonder if the saltlen < 0 should just return 0.

Reply:
Since the implementation has changed, there is no calling to the `rsa_pss_compute_saltlen` function.